src/components/Tag/Tag.tsx

```diff
@@ -18,7 +18,9 @@ export interface TagProps {
 export function Tag({children, disabled = false, onRemove}: TagProps) {
   const i18n = useI18n();
   const className = classNames(disabled && styles.disabled, styles.Tag);
-  const ariaLabel = i18n.translate('Polaris.Tag.ariaLabel', {children});
+  const ariaLabel = i18n.translate('Polaris.Tag.ariaLabel', {
+    children: children || '',
```

Comment:
children could be undefined previously, this ensures that we pass a string into the replacements object as `{children: undefined}` is now a type error

Reply:
It seems like an accessible issue to have a tag without any content or accessibility label. We should probably create an issue for this